src/topp/MascotAdapter.cpp

```diff
@@ -487,10 +487,35 @@ class TOPPMascotAdapter :
 
     if (!mascot_out)
     {
+    // determine number of chunks of size n
+    int size = (experiment.size() - 1) / n + 1;
+ 
+    // create array of vectors to store the chunks
+    std::vector<PeakMap> split_experiment[size];
```

Comment:
is std::vector<PeakMap> a valid type specification?

Reply:
sure. If `vector` is `#include`d, either directly or indirectly :)